tests/coverage/Multiplatform.md

```diff
@@ -0,0 +1,50 @@
+# Where is what, a link list.
```

Comment:
We need this on the website as well IMHO. I almost think it should go in the general README as well XD @mravanelli, @Gastron, @pplantinga, @Adel-Moumen. Ideas?

Reply:
Is it possible to link to specific sections of other Readmes, it should with `.md#Section` right?